src/test/groovy/pl/allegro/tech/opel/OpelEngineMapIntegrationSpec.groovy

```diff
@@ -34,6 +69,7 @@ class OpelEngineMapIntegrationSpec extends Specification {
 
         where:
         input                          || expResult
+        "{x:2, y:5}.x"                 || 2
```

Comment:
add some test when map is assigned to some val. Now we test access to anonymous map and map from context.